volumes/ecocounter/views/create-view-counts.sql

```diff
@@ -5,7 +5,15 @@ CREATE VIEW ecocounter.counts AS (
         counts_unfiltered.volume
     FROM ecocounter.counts_unfiltered
     JOIN ecocounter.flows_unfiltered USING (flow_id)
-    WHERE flows_unfiltered.validated --is true
+    WHERE
+        flows_unfiltered.validated --is true
+        AND NOT EXISTS (
```

Comment:
I think an anti-join is a little less verbose: 
```sql
    --anti join anomalous_ranges
    LEFT JOIN ecocounter.anomalous_ranges USING (flow_id)
    WHERE
        flows_unfiltered.validated --is true
        AND anomalous_ranges.time_range IS NULL
        OR NOT(anomalous_ranges.time_range @> counts_unfiltered.datetime_bin)
```

Reply:
Perhaps less verbose, but I've always found those harder to read. I like the `NOT EXISTS` because, though large, it's a single `WHERE` condition which is conceptually a lot tidier for me.